.github/workflows/tests.yml

```diff
@@ -9,7 +9,7 @@ on:
   # there is a CI failure related to a
   # change in any dependency.
   schedule:
-    - cron: '0 */8 * * *'
+    - cron: '0 0 * * *'
```

Comment:
What is the reason for changing this?

Reply:
was: every 8th hour (I'm guessing 8am and 8pm)
now: every midnight